backend/routers/admin.py

```
@@ -45,18 +45,14 @@ def get_system_stats(db: Session = Depends(get_db)):
-        User.role,
-        User.is_active,
-        func.count(User.id)
-    ).group_by(User.role, User.is_active).all()
-
-    total = sum(c for r, a, c in counts)
-    admins = sum(c for r, a, c in counts if r == UserRole.ADMIN)
-    active = sum(c for r, a, c in counts if a)
+    result = db.query(
+        func.count(User.id).label('total'),
+        func.sum(case((User.role == UserRole.ADMIN, 1), else_=0)).label('admins'),
+        func.sum(case((User.is_active == True, 1), else_=0)).label('active')
```

Ruff (0.15.13)

[error] Avoid equality comparisons to `True`; use `User.is_active:` for truth checks

Replace with `User.is_active`

(E712)